src/Riok.Mapperly/Diagnostics/DiagnosticDescriptors.cs

```diff
@@ -269,4 +269,13 @@ internal static class DiagnosticDescriptors
         DiagnosticCategories.Mapper,
         DiagnosticSeverity.Info,
         true);
+
+    public static readonly DiagnosticDescriptor NotAllEnumValuesMapped = new DiagnosticDescriptor(
+        "RMG034",
+        "Not all enum values in the target enum are mapped",
+        "Not all enum values in the target enum are mapped from {0} to {1}",
+        DiagnosticCategories.Mapper,
+        DiagnosticSeverity.Warning,
```

Comment:
To not introduce a breaking change, I think the default severity should be Info.

Reply:
Agreed, fixed now.